app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+	def index
+	end
+end
```

Comment:
new controller for static homepage [this won't always be static; i am just following the railsbridge docs instructions]